packages/react-native-reanimated/src/createAnimatedComponent/AnimatedComponent.tsx

```diff
@@ -374,6 +376,57 @@ export default class AnimatedComponent
     );
   }
 
+  _configureSharedTransition(useNativeId?: boolean) {
+    if (!getStaticFeatureFlag('SHARED_ELEMENT_TRANSITIONS')) {
+      return;
+    }
+    if (this.props.sharedTransitionTag) {
```

Comment:
This will probably prevent the removal of a shared transition from a component when someone assigns `null` or `undefined`.